vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -434,6 +442,14 @@ func (d *VolumeDriver) Unmount(r volume.UnmountRequest) volume.Response {
 		return volume.Response{Err: ""}
 	}
 
+	// defensive approach to not unmount volumes until refcounting has been successful
+	if d.refCounts.GetSuccess() != true {
+		log.WithFields(
+			log.Fields{"name": r.Name},
+		).Info("Refcounting hasn't been successfull. Skipping unmount request")
```

Comment:
typo: successfull => successful

"Refcounting hasn't been successfull. Skipping unmount request" =>
"Reference count map is not initialized. Unmount is delayed".